connector/src/main/scala/com/microsoft/kusto/spark/datasink/TempStorageCache.scala

```diff
@@ -23,18 +21,19 @@ object TempStorageCache{
   }
 
   private def getNextUri(clusterAlias: String, kcsb: ConnectionStringBuilder): String = {
+    var storageCached = storagesMap.get(clusterAlias)
     // Refresh if 120 minutes have passed since last refresh for this cluster as SAS should be valid for at least 120 minutes
```

Comment:
120 -> storageExpiryMinutes. Consider adding a comment where it is defined that should not be below 120 minutes since SAS...

Reply:
it is already explained